conf/application.conf

```diff
@@ -51,6 +51,7 @@ play.modules.enabled += "uk.gov.hmrc.mongo.play.PlayMongoModule"
 play.modules.enabled += "runtime.IOModule"
 play.modules.enabled += "runtime.SystemClockModule"
 play.modules.enabled += "workers.BalanceRequestUpdateWorkerModule"
+play.modules.enabled += "play.modules.reactivemongo.ReactiveMongoModule"
```

Comment:
Please do not do this - this service uses hmrc-mongo because reactivemongo is deprecated